src/js/Attachment/__test__/onDrop.test.ts

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Attachment from '../index';
+import {createEvent, fireEvent} from '@testing-library/dom';
+
+describe('Unit test for Attachment onDrop event handler', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => { });
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('onDrop is fired successfully for IE', () => {
+    const file = new File([''], 'file1.png', {
+      type: 'image/png',
+    });
+
+    const newFile = new File([''], 'file2.png', {
+      type: 'image/png',
+    });
+
+    const dropHandler = (files: any[]) => {
+      expect(files.length).toEqual(2);
+      expect(files[1]).toBe(newFile);
+    };
```

Comment:
Seem this test is not effected.

Change expect(files.length).toEqual(1); --> expect(files.length).toEqual(344);
Result: test suite is passed
Expect: error is shown